Objects/tupleobject.c

```diff
@@ -11,6 +11,24 @@ class tuple "PyTupleObject *" "&PyTuple_Type"
 
 #include "clinic/tupleobject.c.h"
 
+/* The objects representing an empty tuple.
+   Empty tuples are used all over the place and applications may
+   rely on the fact that an empty tuple is a singleton. */
+
+static struct {
+    PyGC_Head gc_head;
+    PyTupleObject obj;
+} _PyTuple_EmptyStruct = {
+    .gc_head = { .gc = {
+        .gc_refs = (((size_t)_PyGC_REFS_UNTRACKED) << _PyGC_REFS_SHIFT)
```

Comment:
Agree, this is ugly. But there is no `PyTuple_Init`, and even if add it how can we be sure that `_PyTuple_Empty` is not used before `PyTuple_Init`?